### PR TITLE
User registration us11

### LINKED ISSUE #40: User Story 11, User Registration Missing Details
As a visitor
When I visit the user registration page
And I do not fill in this form completely,
I am returned to the registration page
And I see a flash message indicating that I am missing required fields

### PR DESCRIPTION
**Pull request by priyapower**
closes #40 
**Contributors**
- Driver: Priya
- Navigators: Grant & Luke

**What this did**
- Added a flash message during user registration if user fails to enter a required field

**Updates**
- Confirm MVC logic
- Update flash css/html

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -5,9 +5,15 @@ def new
...
-    redirect_to '/profile'
+    user = User.new(user_params)
+    if user.save
+      flash[:success] = "Welcome, #{user.name}"
+      redirect_to '/profile'
+    else
+      flash[:failure] = "You are missing one or more required fields"
+      redirect_to '/register'
+    end
+
```
Comment (LHJE) [inline app/controllers/users_controller.rb:16]: Looks terrific.  This is way more dynamic this way.

File: spec/features/users/registration_spec.rb [hunk 2]
```diff
@@ -28,5 +28,22 @@
...
+      fill_in  :state, with: "CO"
+      fill_in  :zip, with: "82828"
+      fill_in  :email, with: "tombroke@gmail.com"
+      fill_in  :password, with: "hiohio38298"
+      fill_in  :password_confirmation, with: "hiohio38298"
+      click_button "Sign-up"
+      expect(current_path).to eq("/register")
+      expect(page).to have_content("You are missing one or more required fields")
+    end
+
```
Comment (LHJE) [inline spec/features/users/registration_spec.rb:47]: Truly a well made test.

**Review by LHJE — APPROVED** [review]
Everything looks tip top!

File: spec/features/users/registration_spec.rb (hunk 2 above)
Comment (priyapower) [reply spec/features/users/registration_spec.rb:47]: Thank you sir! You guided me!

**Pull request merged**